integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonTestUtils.java

```diff
@@ -670,6 +676,68 @@ public static boolean upgradeAndVerifyOperator(String opNamespace, OperatorParam
     return true;
   }
 
+
+  /**
+   * Install OCI Load Balancer and wait up to five minutes until the External IP is ready.
+   *
+   * @param namespace the namespace in which the Noci Load Balancer will be installed
+   * @param portshttp the http port of oci load balancer
+   * @param clusterName name of WLS cluster
+   * @param domainUID domain UID
+   * @param loadBalancerName service name for OCI Load Balancer
+   */
+  public static void installAndVerifyOCILoadBalancer(
+      String namespace,
+      int portshttp,
+      String clusterName,
+      String domainUID,
+      String loadBalancerName) throws ApiException {
+    Map<String, String> annotations = new HashMap<>();
+    annotations.put("service.beta.kubernetes.io/oci-load-balancer-shape", "400Mbps");
+    Map<String, String> selectors = new HashMap<>();
+    Map<String, String> labels = new HashMap<>();
+    labels.put("loadbalancer", loadBalancerName);
+    selectors.put("weblogic.clusterName", clusterName);
+    selectors.put("weblogic.domainUID",domainUID);
+    List<V1ServicePort> ports = new ArrayList<>();
+    ports.add(new V1ServicePort()
+        .name("http")
+        .port(portshttp)
+        .targetPort(new IntOrString(portshttp))
+        .protocol("TCP"));
+
+    V1Service service = new V1Service()
+        .metadata(new V1ObjectMeta()
+            .name(loadBalancerName)
+            .namespace(namespace)
+            .labels(labels)
+            .annotations(annotations))
+        .spec(new V1ServiceSpec()
+            .ports(ports)
+            .selector(selectors)
+            .sessionAffinity("None")
+            .type("LoadBalancer"));
+    LoggingFacade logger = getLogger();
+    logger.info("Checking service object not null");
+    assertNotNull(service, "Can't create ocilb service, returns null");
+    logger.info("Call to create service object " + loadBalancerName);
```

Comment:
Extraneous logger.info as the statement.

Reply:
fixed